webview/src/plots/components/ribbon/Ribbon.tsx

```diff
@@ -75,6 +76,14 @@ export const Ribbon: React.FC = () => {
           text={`${revisions.length} of ${MAX_NB_EXP}`}
         />
       </li>
+      <li className={styles.buttonWrapper}>
```

Comment:
Example with primary colors:

<img width="554" alt="Screenshot 2023-09-18 at 9 44 19 AM" src="https://github.com/iterative/vscode-dvc/assets/43496356/b18bac50-3bfb-46d5-ae60-272d190a78ad">

Reply:
As per the [guidelines](https://github.com/microsoft/vscode-webview-ui-toolkit/blob/main/src/button/README.md#best-practices) there should only be one primary button. I think Having `Add Plot` as the primary button now makes more sense than the revision selector but up to you.